src/scss/includes/search_form.scss

```diff
@@ -23,7 +23,7 @@
 .search_form__input {
   min-width: 0;
   width: calc(100% - 30px);
-  height: $spacing-xxl-4;
+  height: 46px;
```

Comment:
height seems to be 48px to me, at least on desktop mockups, as it's a value from design system

Reply:
sorry indeed, I forgor to reset my OS zoom before touching that, it's what made me think it's misaligned